source/nodejs/adaptivecards-aaf/src/core.ts

```diff
@@ -134,10 +134,10 @@ export class AdaptiveApplet {
         }
     }
 
-    private createActivityRequest(action: ExecuteAction, context: ActivityInvocationContext): ActivityRequest | undefined {
+    private createActivityRequest(action: ExecuteAction, trigger: ActivityInvocationTrigger): ActivityRequest | undefined {
         if (this.card) {
             let request: ActivityRequest = {
-                context: context,
+                trigger: trigger,
```

Comment:
Sorry I missed that before - per spec, `trigger` should be passed as a property of the `activity.value` object rather than a top level. And its value should be "automatic" or "manual":
![image](https://user-images.githubusercontent.com/1334689/80623394-f696d500-89fe-11ea-9861-a2b29e2ba47f.png)

It might make sense to change the ActivityInvocationTrigger enum values to "Automatic" and "Manual". Would you mind making those changes too?

Reply:
Update the changes